compiler/rustc_middle/src/ty/sty.rs

```diff
@@ -187,12 +187,14 @@ pub enum TyKind<'tcx> {
     /// Looking at the following example, the witness for this generator
     /// may end up as something like `for<'a> [Vec<i32>, &'a Vec<i32>]`:
     ///
-    /// ```rust
+    /// ```ignore HELP (is this supposed to error? the code looks purpose built to trigger E0626 but I can't tell if that's what the doc wants?)
```

Comment:
no, I don't think it's supposed to error - moving the `yield a;` a line earlier seems fine. but @compiler-errors would probably know better than me.